### PR TITLE
Health check endpoint

### PR DESCRIPTION
**Pull request by dylanlott**
add `/health` endpoint that checks database status and returns `503 Service Unavailable` if database down, `200 OK` if everything else is golden. 

Not sure if we want any other metrics or data returned in this, I just went for dead simple to start with. 

Closes https://github.com/Storj/bridge/issues/329

### CODE REVIEW

File: lib/server/routes/health.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return new HealthRouter(options);
+  }
+
+  Router.apply(this, arguments);
+}
+
+inherits(HealthRouter, Router);
+
+HealthRouter.prototype.health = function(req, res) {
+  if (!this.storage) {
```
Comment (braydonf) [inline lib/server/routes/health.js]: We should probably check `this.storage.connection.readyState == 1` to make sure there is a connection.

Comment (dylanlott) [inline lib/server/routes/health.js]: Fixed

File: test/server/routes/buckets.integration.js [hunk 2]
```diff
@@ -53,7 +53,9 @@ describe('BucketsRouter Integration', function() {
 
       let body = '';
 
+
       req.on('response', (res) => {
+        console.log('res', res);
```
Comment (braydonf) [inline test/server/routes/buckets.integration.js]: unneeded log?

**Comment by braydonf** [comment]
LGTM

**Comment by braydonf** [comment]
This matches what's described in https://github.com/Storj/bridge/issues/329 if there are new requirements we can update.

File: lib/server/routes/health.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  Router.apply(this, arguments);
+}
+
+inherits(HealthRouter, Router);
+
+HealthRouter.prototype.health = function(req, res) {
+  if (this.storage.connection.readyState === 1) {
+    res.status(200).send('OK');
```
Comment (phutchins) [inline lib/server/routes/health.js]: Should we be returning here so we don't reach the next res.status?

Comment (braydonf) [inline lib/server/routes/health.js]: Yes, probably. Good catch.

Comment (braydonf) [inline lib/server/routes/health.js]: Added with https://github.com/Storj/bridge/pull/418

**Pull request merged**